### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by Justinnn07**
@stephin007 kindly check and merge😂🤣

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# IPLive
\ No newline at end of file
+# Cric-Live-Scoreboard
+
+> - It is made using ReactJs 
+> - API used => [cric-api](https://www.cricapi.com/)
+> - app deployed at [cric-live](https://cric-liv.web.app/)
```
Comment (stephin007) [inline README.md]: - Use Capital A for app.
- Change the link of the app to the link given in the about section.

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+## Installation 
+
```
Comment (stephin007) [inline README.md:8]: Add a forking step as well before the cloning process.

File: README.md (hunk 1 above)
Comment (Justinnn07) [reply README.md]: okay!

File: README.md (hunk 2 above)
Comment (Justinnn07) [reply README.md:8]: Okay!

**Comment by Justinnn07** [comment]
kindly check and merge @stephin007

File: README.md [hunk 3]
```diff
@@ -31,3 +34,13 @@
 
  > - cd  into the right directory and type `npm run build` or `yarn run build` in the terminal
```
Comment (stephin007) [inline README.md:35]: it should be NPM start/ yarn start and then if the user wants to build the app for production then user should use npm run build/yarn build.
**npm start command missing**

File: README.md [hunk 4]
```diff
@@ -31,3 +34,13 @@
...
 
+## Make sure that you star this repository
+
+<img width="350px" src="https://res.cloudinary.com/practicaldev/image/fetch/s--MJisiJsL--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://thepracticaldev.s3.amazonaws.com/i/2nn6mhp57inp6rdxarzt.png" />
```
Comment (stephin007) [inline README.md]: reduce the width of this image

File: README.md [hunk 5]
```diff
@@ -31,3 +34,13 @@
...
+
+# Author
+## 👤 Stephin Reji
+- Twitter: [reji_setphin](https://twitter.com/reji_stephin)
+- portfolio : [stephinreji.me](https://stephinreji.me)
```
Comment (stephin007) [inline README.md]: Use Capital P

**Comment by Justinnn07** [comment]
now check! @stephin007 😂🤣

**Comment by Justinnn07** [comment]
Now check!!!!!!!!~!

**Comment by Justinnn07** [comment]
CLOSED THISSS

**Pull request closed without merging**